query-algebrizer/src/types.rs

```diff
@@ -35,6 +36,14 @@ pub enum DatomsTable {
     FulltextValues,     // The virtual table mapping IDs to strings.
     FulltextDatoms,     // The fulltext-datoms view.
     AllDatoms,          // Fulltext and non-fulltext datoms.
+    Computed(usize),    // A computed table, tracked elsewhere in the query.
```

Comment:
Strictly speaking it's not an ID, it's an index into the computed tables vector, hence the other chunk of code you asked about.